cmd/util/cmd/compare-cadence-vm/cmd.go

```diff
@@ -108,15 +115,31 @@ func run(_ *cobra.Command, args []string) {
 
 	var blocks []block
 
-	for i := 0; i < flagBlockCount; i++ {
-		header := debug_tx.FetchBlockHeader(blockID, flowClient)
+	if flagBlockCount != 1 {
+		if len(blockIDs) > 1 {
+			log.Fatal().Msg("either provide a single block ID and use --block-count, or provide multiple block IDs and do not use --block-count")
+		}
```

Comment:
I think it would make sense to document this in the description of the command line parameters and not only reveal this information in case of an error.

Reply:
Good idea, improved in [dfef815](https://github.com/onflow/flow-go/pull/7955/commits/dfef815b5512e42dd90e33e7ac39946b3e8db45e)